test/tests/outputs/csv/tests

```diff
@@ -68,6 +68,14 @@
     []
   []
 
+  [all_columns]
+    type = CSVDiff
+    input = 'all_columns_parent.i'
+    csvdiff = 'all_columns_parent_out.csv'
+    requirement = "The system shall allow multiple lines to be output in a CSV file with the same physical time if not identical."
```

Comment:
```suggestion
    requirement = "The system shall allow multiple lines to be output in a CSV file, with the same physical time, if the lines are themselves not otherwise identical."
```
moar words less confusion